src/main/java/uk/gov/hmcts/reform/roleassignment/auditlog/aop/AuditAspect.java

```diff
@@ -47,6 +47,7 @@ public Object audit(ProceedingJoinPoint joinPoint, LogAudit logAudit) throws Thr
             String process = getValue(joinPoint, logAudit.process(), result, String.class);
             String reference = getValue(joinPoint, logAudit.reference(), result, String.class);
             String correlationId = getValue(joinPoint, logAudit.correlationId(), result, String.class);
+            String responseTime = getValue(joinPoint, logAudit.responseTime(), result, String.class);
```

Comment:
you have to calculate time taken to complete process. it can't be pass from anywhere. usually here it will take values from @LogAudit which is using on controller.